privacy_partner_report/controllers/main.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+import json
+from openerp import http
+
+from openerp.addons.web.controllers.main import serialize_exception
+from openerp.addons.web.controllers.main import Reports
+
+
+class ReportsExtended(Reports):
+    # Hack of https://github.com/odoo/odoo/pull/24964
+
+    @http.route()
+    @serialize_exception
+    def index(self, action, token):
+        try:
+            action = json.loads(action)
+            action["datas"] = action["data"]
```

Comment:
If you get a `KeyError` here, `action` is already a `dict`, and call to `super()` will fail.